Cypress/cypress/support/commands.js

```diff
@@ -344,38 +345,48 @@ Cypress.Commands.add(
   (agentName, namespace = "n1", projectId = "") => {
     return cy
       .exec(
-        "kubectl -n litmus get -o jsonpath='{.spec.ports[0].nodePort}' services litmusportal-frontend-service"
+        "kubectl apply -f https://raw.githubusercontent.com/litmuschaos/litmus/master/litmus-portal/litmus-portal-crds.yml",
+        { timeout: 60000 }
       )
+      .then(() => {
```

Comment:
This `.then() `will not be required as we are getting URL from config.

Reply:
@Jonsy13 Updated it